coding-guides/starboard.md

```diff
@@ -0,0 +1,154 @@
+  await message.guild.channels.find('name', starboard).send({ embed });
+}
+```
+
+Now, if you've been following along exactly, you have a working starboard! But if you like a TL;DR, here ya go. 
+
+```js
+module.exports = class {
+  constructor(client) {
+    this.client = client;
+  }
+
+  async run(reaction, user) {
+    const message = reaction.message;
+    if (reaction.emoji.name !== '⭐') return;
+    if (message.author.id === user.id) return message.channel.send(`${user}, you cannot star your own messages.`);
+    if (message.author.bot) return message.channel.send(`${user}, you cannot star bot messages.`);
+    const { starboardChannel } = this.client.settings.get(message.guild.id);
+    const fetch = await message.guild.channels.find('name', starboard).fetchMessages({ limit: 100 });
+    const stars = fetch.find(m => m.embeds[0].footer.text.startsWith('⭐') && m.embeds[0].footer.text.endsWith(message.id));
+    if (stars) {
+      const star = /^\⭐\s([0-9]{1,3})\s\|\s([0-9]{17,20})/.exec(stars.embeds[0].footer.text);
+      const foundStar = stars.embeds[0];
+      const image = message.attachments.size > 0 ? await this.extension(reaction, message.attachments.array()[0].url) : '';
+      const embed = new RichEmbed()
+        .setColor(foundStar.color)
+        .setDescription(foundStar.description)
+        .setAuthor(message.author.name, message.author.displayAvatarURL)
+        .setTimestamp()
+        .setFooter(`⭐ ${parseInt(star[1])+1} | ${message.id}`)
+        .setImage(image);
+      const starMsg = await message.guild.channels.find('name', starboard).fetchMessage(stars.id);
+      await starMsg.edit({ embed });
+    }
+    if (!stars) {
+      if (!message.guild.channels.exists('name', starboard)) throw `It appears that you do not have a \`${starboard}\` channel.`;
+      const image = message.attachments.size > 0 ? await this.extension(reaction, message.attachments.array()[0].url) : '';
+      if (image === '' && message.cleanContent.length < 1) return message.channel.send(`${user}, you cannot star an empty message.`);
+      const embed = new RichEmbed()
+        .setColor(15844367)
+        .setDescription(message.cleanContent)
+        .setAuthor(message.author.tag, message.author.displayAvatarURL)
+        .setTimestamp(new Date())
+        .setFooter(`⭐ 1 | ${message.id}`)
+        .setImage(image);
+      await message.guild.channels.find('name', starboard).send({ embed });
+    }
+  }
+
+  extension(reaction, attachment) {
+    const imageLink = attachment.split('.');
+    const typeOfImage = imageLink[imageLink.length - 1];
+    const image = /(jpg|jpeg|png|gif)/gi.test(typeOfImage);
+    if (!image) return '';
+    return attachment;
+  }
+};
+```
+
+So, all that code up there is only for if a reaction is added. Now, we'll handle if a reaction is removed.
+
+Here, we very closely mimic the messageReactionAdd event, only this time we'll **subtract** 1 from the total amount of stars that are on the embed.
+
+```js
```

Comment:
You should explain what the differences are between `messageReactionAdd` and `messageReactionRemove` and explain what you've actually changed in the code (as well as showing the codeblock)